torch/onnx/utils.py

```diff
@@ -1663,6 +1667,15 @@ def _export(
     return torch_out
 
 
+@_beartype.beartype
+def _add_onnxscript_fn(model_proto: bytes) -> bytes:
+    model = onnx.load_model_from_string(model_proto)
+    if _custom_onnxscript is not None:
+        model.functions.extend(_custom_onnxscript)
```

Comment:
Wondering how I can do this in Cpp. In that case, we don't need onnx for dependency ...

Reply:
checkout https://github.com/pytorch/pytorch/blob/c43033edec898aa5e470d955873d9b175d272be9/torch/csrc/jit/serialization/export.cpp#L776
and
https://github.com/pytorch/pytorch/blob/c43033edec898aa5e470d955873d9b175d272be9/torch/csrc/jit/python/python_ir.cpp#247
If you want you can implement this in C++ and expose through pybind.

Reply:
While I'm curious, does onnx-script depend on onnx? and do we want to add onnx-script dependency to pytorch?